src/quo2/components/avatars/group_avatar.cljs

```diff
@@ -1,32 +1,47 @@
 (ns quo2.components.avatars.group-avatar
   (:require [quo2.components.icon :as icon]
             [quo2.foundations.colors :as colors]
-            [react-native.core :as rn]))
+            [quo2.theme :as theme]
+            [react-native.core :as rn]
+            [react-native.fast-image :as fast-image]))
 
 (def sizes
-  {:icon      {:small  12
-               :medium 16
-               :large  20}
-   :container {:small  20
-               :medium 32
-               :large  48}})
+  {:icon      {:x-small 12
+               :small   16
+               :medium  16
+               :large   20
+               :x-large 32}
+   :container {:x-small 20
+               :small   28
+               :medium  32
+               :large   48
+               :x-large 80}})
 
-;; TODO: this implementation does not support group display picture (can only display default group
-;; icon).
-(defn group-avatar
-  [_]
-  (fn [{:keys [color size]}]
+(defn- group-avatar-internal
+  [_ _]
+  (fn [{:keys [customization-color theme avatar? size]
+        :or   {size                :small
+               customization-color :blue}}]
     (let [container-size (get-in sizes [:container size])
-          icon-size      (get-in sizes [:icon size])]
+          icon-size      (get-in sizes [:icon size])
+          color          (colors/theme-colors
+                          (colors/custom-color customization-color 50)
+                          (colors/custom-color customization-color 60)
+                          theme)]
       [rn/view
        {:width            container-size
         :height           container-size
         :align-items      :center
         :justify-content  :center
         :border-radius    (/ container-size 2)
-        ;:background-color (colors/custom-color-by-theme color 50 60) ; TODO: this is temporary only.
-        ;Issue: https://github.com/status-im/status-mobile/issues/14566
         :background-color color}
-       [icon/icon :i/group
-        {:size  icon-size
-         :color colors/white-opa-70}]])))
+       (if avatar?
```

Comment:
two things `avatar?` suggest a boolean because of the `?`. 

and similiarly in figma it is using `photo` so let's refer to it as that to keep things aligned 
<img width="244" alt="Screenshot 2023-08-09 at 13 14 00" src="https://github.com/status-im/status-mobile/assets/22799766/8a280e9c-407d-428f-9677-97a52c144408">